### DISCUSSION TITLE
Keeps on rebooting after 1st reboot

### DISCUSSION THREAD

**Discussion by sigb1n** [macOS-Dell-Latitude-E7470/macOS-Dell-Latitude-E7470#36] [Q&A] [answered]
I might be doing something wrong but after the 1st reboot, if just keeps on rebooting.  I've copied the exact EFI and I also have the same E7470 specs as yours.  Any help would be appreciated.  Thanks!

**Comment by sigb1n** [comment, marked as answer]
Got this resolved.  Thanks!

**Reply by peretm** [reply]
And how did you get it resolved?

**Reply by kingster85** [reply]
I want to know this too as I’m having the same issue.
